src/rust-crypto/rust-crypto.ts

```diff
@@ -2138,6 +2151,34 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, CryptoEventH
     public async getOwnIdentity(): Promise<RustSdkCryptoJs.OwnUserIdentity | undefined> {
         return await this.olmMachine.getIdentity(new RustSdkCryptoJs.UserId(this.userId));
     }
+    /**
```

Comment:
nit: missing blank line


```suggestion

    /**
```

Reply:
Addressed in c6ff37c2691929c59ac99d1ab7f6e84826dd7f32